routes/volunteers/volunteers.js

```diff
@@ -29,6 +69,65 @@ volunteerRouter.get('/total', async (req, res) => {
   }
 });
 
+volunteerRouter.get('/available', async (req, res) => {
+  try {
+    const { driverOption, eventInterest, searchQuery } = req.query;
+
+    let driverCondition = 'TRUE';
+    let eventCondition = 'TRUE';
+
+    if (driverOption === 'Can Drive') {
+      driverCondition = ' users.can_drive';
+    } else if (driverOption === 'Cannot Drive') {
+      driverCondition = ' NOT users.can_drive';
+    }
+
+    if (eventInterest === 'Distributions') {
+      eventCondition = ' users.distribution_interest';
+    } else if (driverOption === 'Food Running') {
```

Comment:
this should be `eventInterest` but it's getting changed anyways; see my convert to mappings comment above